src/objects/aff/zcl_abapgit_json_handler.clas.abap

```diff
@@ -62,7 +65,24 @@ CLASS zcl_abapgit_json_handler DEFINITION
         RAISING   zcx_abapgit_ajson_error,
       set_abap_language_version
         CHANGING co_ajson TYPE REF TO zcl_abapgit_ajson
-        RAISING  zcx_abapgit_ajson_error.
+        RAISING  zcx_abapgit_ajson_error,
+      "! Get the enum mapping from object handler, as other enums as well
+      set_abap_language_version_des
```

Comment:
I am struggling with the method names starting with `set_`. There are always two methods like `set_abap_language_version` and `set_abap_language_version_des` (one for serialization and one for deserialization). 

Would it make sense to rename them to `serialize_abap_language_vers` and `deserialize_abap_language_vers`?

Alternatively, we can put these methods into two separate local classes (one for serialization one for deserialization).

Reply:
not exactly AFAICS.
actually it is a mapping correct?

what about the general patter:
map2abap_...
map2json_...

Reply:
> what about the general patter:
map2abap_...
map2json_...

Yes, this would be fine for me, too

Reply:
ok, done

json handler in general could get a refactoring. there are object references passed as changing parameters (could be importing actually) and missing unit tests)